px-demo-footer.html

```diff
@@ -47,11 +47,21 @@
           <a href="https://www.ge.com/digital"><img src="[[importPath]]monogram-wdmk.png" alt="GE Monogram" class=""></a>
         </div>
         <div class="demo-footer__list">
-          <ul class="list-inline list-inline--delimited">
-            <li><a href="https://predix.io/legal" class="actionable actionable--secondary" target="_blank">Legal</a></li>
-            <li><a href="https://www.predix.io/support/article/KB0012081" class="actionable actionable--secondary" target="_blank">Contact Us</a></li>
-            <li>Copyright &copy; General Electric Company. All rights reserved.</li>
-          </ul>
+          <template is="dom-if" if="{{_showCookiesLink}}">
+            <ul class="list-inline list-inline--delimited">
+              <li><a on-click="_openCookiesLink" class="actionable actionable--secondary">Cookies</a></li>
+              <li><a href="https://predix.io/legal" class="actionable actionable--secondary" target="_blank">Legal</a></li>
+              <li><a href="https://www.predix.io/support/article/KB0012081" class="actionable actionable--secondary" target="_blank">Contact Us</a></li>
+              <li>Copyright &copy; General Electric Company. All rights reserved.</li>
+            </ul>
+          </template>
+          <template is="dom-if" if="{{!_showCookiesLink}}">
+            <ul class="list-inline list-inline--delimited">
+              <li><a href="https://predix.io/legal" class="actionable actionable--secondary" target="_blank">Legal</a></li>
```

Comment:
Duplication of links is unfortunate, but conditionally rendering _just_ the Cookes link presents styling issues that require reaching into other repos to fix. This was simpler.